packages/core/user-settings/core-user-settings-server-mocks/src/service_contract.mock.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { UserSettingsService } from '@kbn/core-user-settings-server-internal';
+
+export const serviceContractMock = (): jest.Mocked<UserSettingsService> => {
+  return {
+    setup: jest.fn(),
+    start: jest.fn(),
+    stop: jest.fn(),
+  } as unknown as jest.Mocked<UserSettingsService>;
+};
```

Comment:
NIT: if the service only has a `setup` method, then the mock should too. Would avoid the `unknown` cast (which is also error prone if the service's type evolves)

Reply:
Fixed in latest commit!